README.md

```diff
@@ -71,6 +71,47 @@ foo@bar:~$ npm start
 foo@bar:~$ npm run clean
 ```
 
+## Running data_acquisition.py
+If you don't have a local instance of the PostgreSQL table yet, navigate to the `server` directory and run
+```console
+foo@bar:~$ npm start
+```
+to create an empty `job` table.
+
+In a different tab, navigate to the `data_acquisition` directory.
+
+Use [pipenv](https://pipenv.readthedocs.io/en/latest/), a Python dependency manager, to install any missing requirements. Note that we're using Python 3.
```

Comment:
It maybe be worth noting that `pipenv` will install the correct version of Python (assuming it's specified in the Pipfile), though I've also noticed it sometimes has trouble if you use a Python 2 `pip` - so it might be worth noting that users should use a Python 3 `pip`, but otherwise `pipenv` should deal with it.

Reply:
It might also be worth saying the `pipenv` is a dependency _and_ environment manager.